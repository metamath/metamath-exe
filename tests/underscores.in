show statement underscores/alt_html
"cat underscores.html"