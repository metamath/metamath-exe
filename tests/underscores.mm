  $c a $.

  $( This is _italic_ and the command "MM-PA> MINIMIZE__WITH *" and a
     sub_script, a label ~ lab[[``~~__ , and a url: ~ http://a_b__c.com/[[``~~__.
     Doubled chars: [[ `` ~~ __
     <HTML>
     Inside HTML tags:

     This is _italic_ and the command "MM-PA> MINIMIZE__WITH *" and a
     sub_script, a label ~ lab[[``~~__ , and a url: ~ http://a_b__c.com[[``~~__.
     Doubled chars: [[ `` ~~ __
     </HTML> $)
  underscores $a a $.

$( $t
 htmldef "a" as "a";
  althtmldef "a" as "a";
   latexdef "a" as "a"; $)
