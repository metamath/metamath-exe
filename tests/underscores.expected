MM> READ "underscores.mm"
Reading source file "underscores.mm"... 203 bytes
203 bytes were read into the source buffer.
The source has 2 statements; 1 are $a and 0 are $p.
SET EMPTY_SUBSTITUTION was turned ON (allowed) for this database.
No errors were found.  However, proofs were not checked.  Type VERIFY PROOF *
if you want to check them.
MM> Continuous scrolling is now in effect.
MM> Creating HTML file "underscores.html"...
Reading definitions from $t statement of underscores.mm...
1 typesetting statements were read from "underscores.mm".
MM> <!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN"
    "http://www.w3.org/TR/html4/loose.dtd">
<HTML LANG="EN-US">
<HEAD>
<META HTTP-EQUIV="Content-Type" CONTENT="text/html; charset=iso-8859-1">
<META NAME="viewport" CONTENT="width=device-width, initial-scale=1.0">
<STYLE TYPE="text/css">
<!--
img { margin-bottom: -4px }
.r { font-family: "Arial Narrow";
     font-size: x-small;
   }
.i { font-family: "Arial Narrow";
     font-size: x-small;
     color: gray;
   }
-->
</STYLE>

<TITLE>underscores - Metamath Test Page</TITLE>
<LINK REL="shortcut icon" HREF="favicon.ico" TYPE="image/x-icon">
</HEAD>
<BODY BGCOLOR="#FFFFFF">
<TABLE BORDER=0 CELLSPACING=0 CELLPADDING=0 WIDTH="100%">
  <TR>
    <TD ALIGN=LEFT VALIGN=TOP WIDTH="25%"><A HREF=
    "mmset.html"><IMG SRC="mm.gif"
      BORDER=0
      ALT="MTP Home"
      TITLE="MTP Home"
      HEIGHT=32 WIDTH=32 ALIGN=TOP STYLE="margin-bottom:0px"></A>
    </TD>
    <TD ALIGN=CENTER COLSPAN=2 VALIGN=TOP><FONT SIZE="+3" COLOR="#006633"><B>
Metamath Test Page
      </B></FONT></TD>
    <TD ALIGN=RIGHT VALIGN=TOP WIDTH="25%">
      <FONT SIZE=-1 FACE=sans-serif>
      <A HREF="underscores.html">
      &lt; Wrap</A>&nbsp;&nbsp;
      <A HREF="underscores.html">Wrap &gt;</A>
      </FONT><FONT FACE=sans-serif SIZE=-2>
      <BR><A HREF="mmtheorems1.html#underscores">Nearby theorems</A>
      </FONT>
    </TD>
  </TR>
  <TR>
    <TD COLSPAN=2 ALIGN=LEFT VALIGN=TOP><FONT SIZE=-2
      FACE=sans-serif>
      <A HREF="../mm.html">Mirrors</A>&nbsp; &gt;
      &nbsp;<A HREF="../index.html">Home</A>&nbsp; &gt;
      &nbsp;<A HREF="mmset.html">MTP Home</A>&nbsp; &gt;
      &nbsp;<A HREF="mmtheorems.html">Th. List</A>&nbsp; &gt;
      &nbsp;underscores
      </FONT>
    </TD>
    <TD COLSPAN=2 ALIGN=RIGHT VALIGN=TOP>
      <FONT SIZE=-2 FACE=sans-serif>

      </FONT>
    </TD>
  </TR>
</TABLE>
<HR NOSHADE SIZE=1>
<CENTER><B><FONT SIZE="+1">Syntax Definition <FONT
COLOR="#006633">underscores</FONT></FONT></B>&nbsp;<SPAN CLASS=r
STYLE="color:#FB0000">1</SPAN></CENTER>
<CENTER><TABLE><TR><TD ALIGN=LEFT><B>Description: </B>This is <I>italic</I> and
the command &quot;MM-PA&gt; MINIMIZE_WITH *&quot; and a
     sub<SUB><FONT SIZE="-1">script</FONT></SUB>.</TD></TR></TABLE></CENTER>

<CENTER><TABLE BORDER CELLSPACING=0 BGCOLOR="#EEFFFA"
SUMMARY="Assertion">
<CAPTION><B>Assertion</B></CAPTION>
<TR><TH>Ref
</TH><TH>Expression</TH></TR>
<TR ALIGN=LEFT><TD><FONT COLOR="#006633"><B>underscores</B></FONT></TD><TD>
<SPAN >a</SPAN></TD></TR>
</TABLE></CENTER>


<HR NOSHADE SIZE=1>
</TABLE></CENTER>
<TABLE BORDER=0 WIDTH="100%">
<TR><TD WIDTH="25%">&nbsp;</TD>
<TD ALIGN=CENTER VALIGN=BOTTOM>
<FONT SIZE=-2 FACE=sans-serif>
Copyright terms:
<A HREF="../copyright.html#pd">Public domain</A>
</FONT></TD><TD ALIGN=RIGHT VALIGN=BOTTOM WIDTH="25%">
<FONT SIZE=-2 FACE=sans-serif>
<A HREF="http://validator.w3.org/check?uri=referer">
W3C validator</A>
</FONT></TD></TR></TABLE>
</BODY></HTML>
